src/soc/ibm/power9/fir.c

```diff
@@ -0,0 +1,195 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
Almost each component has it's own set of FIR, having this in separate file is misleading. Please move it to occ.c. Same goes for fir.h.